cmd/kube-apiserver/app/options/options.go

```diff
@@ -231,7 +231,8 @@ func (s *ServerRunOptions) Flags() (fss cliflag.NamedFlagSets) {
 
 	fs.StringVar(&s.ServiceClusterIPRanges, "service-cluster-ip-range", s.ServiceClusterIPRanges, ""+
 		"A CIDR notation IP range from which to assign service cluster IPs. This must not "+
-		"overlap with any IP ranges assigned to nodes or pods. Max of two dual-stack CIDRs is allowed.")
+		"overlap with any IP ranges assigned to nodes or pods. Max of two dual-stack CIDRs is allowed. "+
+		"The largest supported service subnet is /12 for IPv4 and /112 for IPv6.")
```

Comment:
If we're going to list specific numbers here, then a recommended size seems more useful than a max size. (which, I think, is /16 for IPv4 and /112 for IPv6)

Reply:
Something like below? @danwinship 

`The largest supported service subnet is /12 for IPv4 and /112 for IPv6.`
`The recommended service subnet is /16 for IPv4 and /112 for IPv6.`

Reply:
Or something like
`The recommended service subnet is /16 for IPv4(no larger than 12/) and /112 for IPv6(no larger than /112).`